crates/levm/src/memory.rs

```diff
@@ -10,6 +10,10 @@ impl Memory {
         Self { data: Vec::new() }
     }
 
+    pub fn new_from_vec(data: Vec<u8>) -> Self {
+        Self { data }
+    }
```

Comment:
Can't we implement `From<Vec<u8>> for `Memory``?

Reply:
We can, done!!